src/index.html

```diff
@@ -0,0 +1,15 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <link rel="stylesheet" href="style.css">
+  <title>Colored-Blocks</title>
+</head>
+<body>
+  <div class="block__red"></div>
+  <div class="block__green"></div>
+  <div class="block__blue"></div>
```

Comment:
you try to use BEM, it is great, but color should be modifier and should be written `class="block block--red"`

Reply:
thank for help, it's really more useful then I did